.travis.yml

```diff
@@ -201,6 +201,9 @@ script:
   - $make PLATFORM=ls-ls1021atwr
   - $make PLATFORM=ls-ls1021aqds
 
+  # Xilinx Zynq 7000
```

Comment:
Without knowing too much about Xilinx platforms, is "7000" and "zc702" considered as two different names for the same hardware?